src/Compilers/VisualBasic/BasicAnalyzerDriver/VisualBasicDeclarationComputer.vb

```diff
@@ -100,12 +116,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         ComputeDeclarationsCore(model, decl, shouldSkip, getSymbol, builder, newLevel, cancellationToken)
                     Next
                     Dim propertyInitializers = GetInitializerNodes(t.PropertyStatement)
-                    builder.Add(GetDeclarationInfo(model, node, getSymbol, propertyInitializers, cancellationToken))
+                    Dim attributes = GetAttributes(t.PropertyStatement.AttributeLists)
```

Comment:
>Dim attributes = GetAttributes(t.PropertyStatement.AttributeLists) [](start = 20, length = 66)

Aren't we doing this below?

Reply:
[https://github.com/dotnet/roslyn/issues/23347](https://github.com/dotnet/roslyn/issues/23347)

---
In reply to: [152418194](https://github.com/dotnet/roslyn/pull/23318#discussion_r152418194) [](ancestors = 152418194)